f default response channel

Rikshy opened the pull request.

Rikshy committed:
created feature draft

Rikshy committed:
send nudes

Rikshy committed:
lint

Rikshy committed:
....

Rikshy committed:
random fixes

Rikshy committed:
send less nudes

Rikshy committed:
lint

Rikshy committed:
fuck you

Rikshy committed:
like this?

karaktaka commented on NerdyPy/NerdyPy.py:
```diff
@@ -12,10 +12,12 @@
 import configparser
 from pathlib import Path
 from datetime import datetime
+
```
Warum ist da ein Leerschritt?

Rikshy replied:
War dir Zeilenumbruch zu Mainstream? ^^

Ich hab kA, habe die imports nicht manuell ausgeführt.

Ist das jetzt ne Aufforderung das zu Ändern?

Rikshy replied:
Wenn man auf die Datei in PyCharm "Rechtsklick -> Reformate Code -> Optimize Imports" macht wird das alles automatisch so formatiert und ist dementsprechen das standard verhalten für neue Imports

karaktaka commented on NerdyPy/models/default_channel.py:
```diff
@@ -0,0 +1,29 @@
+""" uhm """
```
Kannst du da noch einen richtigen Kommentar einfügen?

karaktaka commented on NerdyPy/modules/management.py:
```diff
@@ -1,9 +1,14 @@
+from datetime import datetime
+
```
Warum ist hier ein Leerschritt?

karaktaka commented on NerdyPy/modules/management.py:
```diff
@@ -1,9 +1,14 @@
 import discord
 import utils.format as fmt
+from models.default_channel import DefaultChannel
 
```
Warum ist hier ein Leerschritt?

karaktaka commented on NerdyPy/modules/management.py:
```diff
@@ -1,9 +1,14 @@
 
 from utils.checks import is_botmod
 from discord.ext.commands import Cog, command, group, check
 
```
Warum ist hier ein Leerschritt?

Rikshy replied:
Du kannst übrigens mehrere Zeilen markieren und diese dann kommentieren.

Rikshy committed:
https://www.duden.de/rechtschreibung/Leerschritt

> Abstand, der sich beim Maschinenschreiben durch einen Anschlag der Leertaste ergibt

Rikshy committed:
richtiger kommentar
